src/xhr.js

```diff
@@ -16,6 +16,45 @@ const {
   mergeOptions
 } = videojs;
 
+const callbackWrapper = function(request, error, response, callback) {
```

Comment:
This code is unchanged, I just moved it into its own function so that container-request can use it.